README.md

```diff
@@ -144,7 +160,7 @@ You need to restrict access to your jenkins endpoint (load balancer). Here's how
 1. Using command line as below:
 
 ```
-npm run cdk synth OpenSearch-CI-Dev -- -c useSsl=false -c runWithOidc=false -c serverAccessType=ipv4 -c restrictServerAccessTo=10.10.10.10/32
```

Comment:
The runWithOidc is only for federated here? and you dont need authType for synth comment?

Reply:
Nope. removed the runWithOidc instances in readme